packages/compiler-core/src/transform.ts

```diff
@@ -59,7 +59,7 @@ export type DirectiveTransform = (
   context: TransformContext,
   // a platform specific compiler can import the base transform and augment
   // it by passing in this optional argument.
-  augmentor?: (ret: DirectiveTransformResult) => DirectiveTransformResult
```

Comment:
this was probably correct, let's keep the old version

Reply:
Aha! augment -> augmentor, That's indeed confusing 😅